openfecli/commands/input.yaml

```diff
@@ -0,0 +1,43 @@
+# things behind a "method" key have to match up to a known thing in openfe
+
+network:
+  method: plan_radial_network
+  scorer: default_lomap_score
+
+# Question:
+# use settings here, to match Protocol
```

Comment:
I like the "settings" nest, since a) allows for further expansion for non-settings keys if they are ever needed, b) it's just visually pleasing to look at.

Not a super strong vote though.

Reply:
+1 on settings nesting